vnext/Microsoft.ReactNative.SharedManaged/JSValueGenerator.cs

```diff
@@ -437,7 +437,8 @@ public static Expression SetProperty(this Expression instance, PropertyInfo prop
     {
       argTypes = parameters.Take(parameters.Length - 1).Select(p => p.ParameterType).ToArray();
       args = argTypes.Select(t => Variable(t, out _)).ToArray();
-      promiseResultType = parameters[parameters.Length - 1].ParameterType.GetGenericArguments()[0];
```

Comment:
nit: LINQ gives Enumerable an extension method to get the last element. This might be a bit cleaner than array[array.Length - 1]

https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.last?view=netframework-4.8

Reply:
Generally, extensions are slower. Anyway, in my latest iteration I rolled back changes in this file.

---
In reply to: [375584058](https://github.com/microsoft/react-native-windows/pull/4041#discussion_r375584058) [](ancestors = 375584058)